### ISSUE TITLE
Usersnap Feedback - Roll back French version of this case to DATE TBC (more info to follow)

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#1319]
**Sender**: jesi.carson@gmail.com
**Comment**: Roll back French version of this case to DATE TBC (more info to follow)
[Open #1007 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/1007)

<a href='https://usersnappublic.s3.amazonaws.com/2020-12-16/18-34/150c86fc-1e56-4aff-a3e0-710cc9ff3e88.png'>![Usersnap Feedback - Roll back French version of this case to DATE TBC (more info to follow)](https://usersnappublic.s3.amazonaws.com/2020-12-16/18-34/9ac5b365-0f0f-40ad-b6a4-aeb3c24f0a2a.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2020-12-16/18-34/150c86fc-1e56-4aff-a3e0-710cc9ff3e88.png'>Download original image</a>

**Browser**: Chrome 86 (Macintosh)
**Referer**: [https://participedia.net/case/7180](https://participedia.net/case/7180)
**Screen size**: 1280 x 800 **Browser size**: 1280 x 698

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
From Antonin: I made some modifications to the English translation of this case since the translated text was somewhat off. My problem is that the English version hasn’t been modified while the French version (the original entry) has been replaced by the modified English text. Is there a way to fix this issue and go back to the original text in French? In the meantime, I changed the settings of the entry to make it private.
